pkg/apis/navigator/types.go

```diff
@@ -89,10 +89,11 @@ type ElasticsearchClusterList struct {
 }
 
 type ElasticsearchClusterSpec struct {
+	Version   string
```

Comment:
I'd be inclined to make this type `*semver.Version` ?

Reply:
I'm not sure we can just make this some other struct. The Golang JSON parser would fail because `string` is not a structure, which it would expect the field to be.

Reply:
That said, UnmarshalJSON etc. is defined so *maybe* it would. I'll take a look. Depending on external types in our apis can get pretty prickly though - if we update our dependency it could cause breaking API changes (and we don't 'control' the downstream code, which makes it harder to fix/manage)